packages/@adobe/spectrum-css-temp/components/buttongroup/index.css

```diff
@@ -15,23 +14,32 @@ governing permissions and limitations under the License.
 }
 
 .spectrum-ButtonGroup {
-  display: flex;
+  display: inline-flex;
+  /* Added this so it would wrap */
+  flex-wrap: wrap;
+  align-items: flex-end;
 
   .spectrum-Button,
-  .spectrum-ActionButton {
+  .spectrum-ActionButton,
+  .spectrum-Tool {
     flex-shrink: 0;
   }
 
+  > .spectrum-Rule--vertical {
```

Comment:
spectrum rule shouldn't be in this file? nor should tool, action button, or button. They'll have different names than their counterparts in the the button css file and the divider css file

Reply:
kk removed